Exception is thrown with empty message when trp_hourFormat attribute has invalid value.

If invalid value is passed to enum property, we will get a build error:
`AAPT: error: '420' is incompatible with attribute trp_hourFormat (attr) enum [FORMAT_SYSTEM=0, FORMAT_12=1, FORMAT_24=2]`

Code: 
```
<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    xmlns:tools="http://schemas.android.com/tools"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    tools:context=".MainActivity">

    <nl.joery.timerangepicker.TimeRangePicker
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        app:trp_hourFormat="420" />

</LinearLayout>
```

But, nevertheless, if we pass a reference to integer with invalid value, we won't get any build error and code compiles successfully:
_ints.xml_
```
<resources>
    <integer name="some_invalid_value">3</integer>
</resources>
```
_activity_main.xml_
```
<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    xmlns:tools="http://schemas.android.com/tools"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    tools:context=".MainActivity">

    <nl.joery.timerangepicker.TimeRangePicker
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        app:trp_hourFormat="@integer/some_invalid_value" />

</LinearLayout>
```

In runtime, we will get exception with no message:
```
     Caused by: java.lang.IllegalArgumentException
        at nl.joery.timerangepicker.TimeRangePicker$HourFormat$Companion.fromId(TimeRangePicker.kt:1160)
        at nl.joery.timerangepicker.TimeRangePicker.initAttributes(TimeRangePicker.kt:147)
        at nl.joery.timerangepicker.TimeRangePicker.<init>(TimeRangePicker.kt:114)
        at nl.joery.timerangepicker.TimeRangePicker.<init>(TimeRangePicker.kt:39)
        at nl.joery.timerangepicker.TimeRangePicker.<init>(TimeRangePicker.kt)
```

Library tries to handle such cases but unsuccessfully:
https://github.com/Droppers/TimeRangePicker/blob/a5c73dbbcb87dcc6b5d1c1f1f5b57e5c330bccd9/timerangepicker/src/main/java/nl/joery/timerangepicker/TimeRangePicker.kt#L135-L140
https://github.com/Droppers/TimeRangePicker/blob/a5c73dbbcb87dcc6b5d1c1f1f5b57e5c330bccd9/timerangepicker/src/main/java/nl/joery/timerangepicker/TimeRangePicker.kt#L1081-L1086


Method `fromId` never returns null, but throws exception with empty message on invalid value, but `initAttributes` expects that `fromId` returns null and assignes default value (`HourFormat.FORMAT_12`) if returned enum value is null.

So, what's expected behavior? To leave default value, or to throw exception. If the last one, it's better to throw exception with some message like 'Invalid attribute value(trp_hourFormat=_value_)'

--- a/timerangepicker/src/main/java/nl/joery/timerangepicker/TimeRangePicker.kt
+++ b/timerangepicker/src/main/java/nl/joery/timerangepicker/TimeRangePicker.kt
@@ -137,12 +137,13 @@ class TimeRangePicker @JvmOverloads constructor(
             context.obtainStyledAttributes(attributeSet, R.styleable.TimeRangePicker, 0, 0)
         try {
             // Time
+            // Sets public property to determine 24 / 12 format automatically
             hourFormat = HourFormat.fromId(
                 attr.getInt(
                     R.styleable.TimeRangePicker_trp_hourFormat,
                     _hourFormat.id
                 )
-            ) ?: _hourFormat // Sets public property to determine 24 / 12 format automatically
+            )
             _angleStart = minutesToAngle(
                 attr.getInt(
                     R.styleable.TimeRangePicker_trp_endTimeMinutes,
@@ -237,7 +238,7 @@ class TimeRangePicker @JvmOverloads constructor(
                     R.styleable.TimeRangePicker_trp_clockFace,
                     _clockFace.id
                 )
-            ) ?: _clockFace
+            )
 
             _clockLabelSize = attr.getDimensionPixelSize(
                 R.styleable.TimeRangePicker_trp_clockLabelSize,
@@ -1172,7 +1173,7 @@ class TimeRangePicker @JvmOverloads constructor(
         FORMAT_24(2);
 
         companion object {
-            fun fromId(id: Int): HourFormat? {
+            fun fromId(id: Int): HourFormat {
                 for (f in values()) {
                     if (f.id == id) return f
                 }
@@ -1186,7 +1187,7 @@ class TimeRangePicker @JvmOverloads constructor(
         SAMSUNG(1);
 
         companion object {
-            fun fromId(id: Int): ClockFace? {
+            fun fromId(id: Int): ClockFace {
                 for (f in values()) {
                     if (f.id == id) return f
                 }

